main.tf

```diff
@@ -0,0 +1,3 @@
+provider "aws" {
+  region = "us-east-1"
+}
```

Comment:
I think a module shouldn't have a `provider` block, but if it's necessary, maybe we should let the `region` argument be an input variable?